src/ControlSystem/ControlErrors/Size/Error.hpp

```diff
@@ -63,26 +60,20 @@ struct ErrorDiagnostics {
  * \param predictor_delta_radius ZeroCrossingPredictor for the difference
  *        in radius between the horizon and the excision boundary.
  * \param time the current time.
+ * \param control_error_delta_r the control error for the DeltaR state. This is
+ *        used in other states as well.
+ * \param dt_lambda_00 the time derivative of the map parameter lambda_00
  * \param apparent_horizon the current horizon in frame Frame.
  * \param excision_boundary a Strahlkorper representing the excision
  *        boundary in frame Frame.  Note that the excision boundary is assumed
  *        to be a sphere in the grid frame.
- * \param grid_frame_excision_boundary_radius the radius of the (assumed
- *        spherical) excision boundary in the grid frame.
- * \param time_deriv_apparent_horizon Time derivative of apparent_horizon,
- *        where the time derivative is taken in frame Frame.  That is, this
- *        should simply be lim dt->0
- *        (apparent_horizon(t+dt)-apparent_horizon(t))/dt with no advective
- *        terms.
```

Comment:
Please add these dox to the new function `control_error_delta_r`.  In particular, we shouldn't lose the information about what frame the time derivative is taken in, and that the excision bdry is assumed spherical in the grid frame.